src/utils.ts

```diff
@@ -11,6 +11,19 @@ export const TWO = new BigNumber(2);
 export const TEN = new BigNumber(10);
 export const MAX_U256 = TWO.pow(255).minus(1);
 
+export interface TokenInfo {
+  readonly chainId: number;
+  readonly address: string;
+  readonly name: string;
+  readonly decimals: number;
+  readonly symbol: string;
+  readonly logoURI?: string;
+  readonly tags?: string[];
+  readonly extensions?: {
+    readonly [key: string]: string | number | boolean | null;
+  };
```

Comment:
Probably don't even need these.

Reply:
Agreed. At least not for now. No idea what extensions are for tbh.

Reply:
My only concern is if we remove them and then the lists we fetch via URL won't parse because they contain more info than the type contains.

Reply:
Yup. Also it's cleaner if the interface represents the jsons data correctly even if we don't need everything.

Reply:
So I may leave this in for now.